ref_builder/cli/events.py

```diff
@@ -0,0 +1,39 @@
+import sys
+from pathlib import Path
+
+import click
+
+from ref_builder.cli.utils import pass_repo
+from ref_builder.console import print_event, print_event_as_json
+from ref_builder.options import path_option
+from ref_builder.repo import Repo, locked_repo
+
+
+@click.group(name="events")
+@path_option
+@click.pass_context
+def events(ctx: click.Context, path: Path) -> None:
+    """Read events."""
+    ctx.obj = ctx.with_resource(locked_repo(path))
+
+
+@events.command(name="get")
+@click.argument("IDENTIFIER", type=int)
+@click.option(
+    "--as-json",
+    "--json",
```

Comment:
This allows you to use options that might collide with reserved words or module names.
```suggestion
    "--json",
    "json_",
```